sw/device/lib/dif/dif_clkmgr_unittest.cc

```diff
@@ -33,10 +33,11 @@ class JitterEnableTest : public ClkMgrTest {};
 // dif_clkmgr_jitter_set_enabled doesn't perform a read, just a write.
 TEST_F(JitterEnableTest, SetEnabled) {
   // Disable jitter.
-  EXPECT_WRITE32(CLKMGR_JITTER_ENABLE_REG_OFFSET, kMultiBitBool4False);
+  EXPECT_READ32(CLKMGR_JITTER_ENABLE_REG_OFFSET, kMultiBitBool4False);
   EXPECT_DIF_OK(dif_clkmgr_jitter_set_enabled(&clkmgr_, kDifToggleDisabled));
```

Comment:
This looks a bit strange to me: as far as I know, it's no longer possible to disable jitter after it has been enabled. Do we just need to update the comment on line 35?

Reply:
Thanks for your feedback @rswarbrick ! This here is just the DIF unittest. It doesn't take into account the behavior of the hardware (the jitter not being disablable once enabled). What this checks is whether the DIF functions when called with different arguments generate the correct number of reads and writes. And the test does two calls to the corresponding DIF function: once to enable the jitter (a read and a write) and once to disable it (one read only).

Reply:
You are right, the comment is misleading.

Reply:
Ah, now I understand. Sorry @rswarbrick , I've misread your comment earlier. I'll update this with the next push.